packages/sdk/react-client/src/client/ClientContext.test.tsx

```diff
@@ -82,23 +88,19 @@ describe('ClientProvider', () => {
   let client: Client;
 
   beforeEach(async () => {
+    // TODO(wittjosiah): Use test builder to avoid warnings.
     client = new Client({ services: fromHost() });
     await client.initialize();
     await client.halo.createIdentity({ displayName: 'test-user' });
   });
 
-  afterEach(async () => {
-    await client.destroy();
-  });
-
   test('Renders with children', async () => {
     render(
       <ClientProvider client={client}>
         <TestComponent />
       </ClientProvider>
     );
 
-    // TODO(wittjosiah): This no longer works. Because the client status becomes active during initialization?
```

Comment:
The `afterEach` was breaking this. The ClientProvider destroys client on unmount so it's unneeded.